minifi-api/include/minifi-cpp/core/ContentRepository.h

```diff
@@ -38,7 +38,8 @@ namespace org::apache::nifi::minifi::core {
 /**
  * Content repository definition that extends StreamManager.
  */
-class ContentRepository : public virtual core::CoreComponent, public virtual StreamManager<minifi::ResourceClaim>, public virtual utils::EnableSharedFromThis, public virtual core::RepositoryMetricsSource {
+class ContentRepository : public virtual core::CoreComponent, public virtual StreamManager<minifi::ResourceClaim>,
+                          public virtual utils::EnableSharedFromThis, public virtual core::RepositoryMetricsSource {
```

Comment:
continuations shouldn't be aligned. I'd either keep the long line, or use two levels of indentation (4 spaces) as continuation indentation.

Reply:
Updated in https://github.com/apache/nifi-minifi-cpp/pull/1977/commits/7d687ae5c9b52b7028aafe5f5d58f587440ef2dc